VRDR/IJEMortality.cs

```diff
@@ -1230,7 +1226,7 @@ public string LIMITS
         {
             get
             {
-                switch (record.ResidenceWithinCityLimits)
```

Comment:
We have code that effectively takes a "Y" or "N", converts it to a boolean, and then converts it back to a "Y" or an "N"...  except that there can be non-Y-N results and those do actually change value. In this case it's converting an "NA" to a "U"... these aren't exactly the same semantically, but I'm not sure we can do better. Just noting, I don't *think* there's an action here.

Reply:
We should probably inform AMS of the discrepancy between U (IJE) and NA (the only code other than Y or N in the ValueSet he selected for the profile).